src/ape/managers/_contractscache.py

```diff
@@ -577,15 +577,18 @@ def get(
                     if fetch_from_explorer
                     else None
                 )
+                contract_type_to_cache = None
+
                 if proxy_contract_type:
                     contract_type_to_cache = _get_combined_contract_type(
                         proxy_contract_type, proxy_info, implementation_contract_type
```

Comment:
Here was also a problem, because it would try and merge `implementation_contract_type = None` w/ `proxy_contract_type`

Reply:
ok i think i got it fixed now